### ISSUE TITLE
GroMoPo MWE is down

### ISSUE THREAD

**Issue by samzipper** [Gromopo/GroMoPo#27]
The streamlit app (both [main](https://share.streamlit.io/juancastilla/gromopo/main/streamlit/GroMoPo_app.py) and [develop](https://share.streamlit.io/juancastilla/gromopo/develop/streamlit/GroMoPo_app.py) versions) is not currently functional. This error appears for me in both Firefox and Chrome:
```
AttributeError: This app has encountered an error. The original error message is redacted to prevent data leaks. Full error details have been recorded in the logs.
Traceback:

File "/home/appuser/venv/lib/python3.7/site-packages/streamlit/script_runner.py", line 354, in _run_script
    exec(code, module.__dict__)
File "/app/gromopo/streamlit/GroMoPo_app.py", line 129, in <module>
    all_gdf,shp_dir = load_shp(stdir,epsg=epsg)
File "/home/appuser/venv/lib/python3.7/site-packages/streamlit/legacy_caching/caching.py", line 543, in wrapped_func
    return get_or_create_cached_value()
File "/home/appuser/venv/lib/python3.7/site-packages/streamlit/legacy_caching/caching.py", line 527, in get_or_create_cached_value
    return_value = func(*args, **kwargs)
File "/app/gromopo/streamlit/GroMoPo_app.py", line 100, in load_shp
    if all_gdf.crs.to_epsg() != epsg: # convert crs, if needed
```

I would like to provide a link to the MWE in the CUAHSI proposal (due next Friday) so hopefully it is a quick fix! If not, I will try to find a screenshot of the old version.

**Comment by juancastilla** [comment]
Hi @samzipper apologies for the delay in responding, was caught up with other projects. I will have a look at the problem which might be related to an update in Streamlit (the GroMoPo code has not changed so its the only possible explanation).

**Comment by rreinecke** [comment]
I have fixed that in #28
